lib/jwt_signed_request/verify.rb

```diff
@@ -70,7 +70,8 @@ def claims
 
     def verified_request?
       claims['method'].to_s.downcase == request.request_method.downcase &&
-        claims['path'] == request.fullpath &&
+        parsed_claims_path.path == request.path &&
```

Comment:
hmm, maybe each of these tests should raise a different error, so that we can tell which part has failed?

Reply:
I'd +1 that, I think it will make any future problems much easier to debug if we can tell which part of the verification is failing from the error.

Reply:
I think it belongs in a separate PR